src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,16 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(str -> str.split(","))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
+                .mapToInt(Integer::intValue)
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list: "
+                        + numbers.toString()));
```

Comment:
Unnecessary toString call 
```suggestion
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: "
                        + numbers.));
```